src/builtin.c

```diff
@@ -1879,6 +1879,32 @@ BINOPS
 #undef LIBM_DD
 #undef LIBM_DA
 
+// This is a hack to make last(empty) yield no output values without using boxing.
```

Comment:
Maybe should say "last(g) where g does not output any value" so that it does not sound like it's a special case for exactly argument "empty"?

Reply:
I fixed the comment.